spinnaker/spinnaker_system/google_http_lb_upsert_test.py

```diff
@@ -134,31 +140,34 @@ def scenario(self):
   def test_c_upsert_min_lb(self):
     self.run_test_case(self.scenario.upsert_min_load_balancer())
 
-  def test_p_upsert_full_lb(self):
+  def test_d_upsert_full_lb(self):
     self.run_test_case(self.scenario.upsert_full_load_balancer())
 
-  def test_s_change_hc(self):
+  def test_e_add_security_group(self):
+    self.run_test_case(self.scenario.add_security_group())
+
```

Comment:
I think having one more dimension here could be helpful, but we would have to provide pretty detailed examples on how they work. Could we use a decorator rather than encode the phases into the name? I actually think using a decorator for both would be useful -- something like `@phase('create')` or `@create_phase`, etc, then an ordering like `@order(1)`.